src/authenticationservices.cs

```diff
@@ -743,4 +778,148 @@ interface ASWebAuthenticationSessionWebBrowserSessionManager {
+	[BaseType (typeof (ASAccountAuthenticationModificationRequest))]
+	[DisableDefaultCtor]
+	interface ASAccountAuthenticationModificationReplacePasswordWithSignInWithAppleRequest {
+
+		[Export ("initWithUser:serviceIdentifier:userInfo:")]
+		IntPtr Constructor (string user, ASCredentialServiceIdentifier serviceIdentifier, [NullAllowed] NSDictionary userInfo);
+
+		[Export ("user")]
+		string User { get; }
+
+		[Export ("serviceIdentifier")]
+		ASCredentialServiceIdentifier ServiceIdentifier { get; }
+
+		[NullAllowed, Export ("userInfo")]
+		NSDictionary UserInfo { get; }
+	}
+
+	[iOS (14,0)]
+	[NoWatch, NoTV, NoMac]
+	[Unavailable (PlatformName.MacCatalyst)][Advice ("This API is not available when using Catalyst on macOS.")]
+	[BaseType (typeof (ASAccountAuthenticationModificationRequest))]
+	[DisableDefaultCtor]
+	interface ASAccountAuthenticationModificationUpgradePasswordToStrongPasswordRequest {
+
+		[Export ("initWithUser:serviceIdentifier:userInfo:")]
+		IntPtr Constructor (string user, ASCredentialServiceIdentifier serviceIdentifier, [NullAllowed] NSDictionary userInfo);
+
+		[Export ("user")]
+		string User { get; }
+
+		[Export ("serviceIdentifier")]
+		ASCredentialServiceIdentifier ServiceIdentifier { get; }
+
+		[NullAllowed, Export ("userInfo")]
+		NSDictionary UserInfo { get; }
+	}
+
+	[iOS (14,0)]
+	[NoWatch, NoTV, NoMac]
+	[Unavailable (PlatformName.MacCatalyst)][Advice ("This API is not available when using Catalyst on macOS.")]
+	[BaseType (typeof (UIViewController))]
+	interface ASAccountAuthenticationModificationViewController {
+
+		[Export ("extensionContext", ArgumentSemantic.Strong)]
+		ASAccountAuthenticationModificationExtensionContext ExtensionContext { get; }
+
+		[Export ("convertAccountToSignInWithAppleWithoutUserInteractionForServiceIdentifier:existingCredential:userInfo:")]
+		void ConvertAccountToSignInWithAppleWithoutUserInteraction (ASCredentialServiceIdentifier serviceIdentifier, ASPasswordCredential existingCredential, [NullAllowed] NSDictionary userInfo);
+
+		[Export ("prepareInterfaceToConvertAccountToSignInWithAppleForServiceIdentifier:existingCredential:userInfo:")]
+		void PrepareInterfaceToConvertAccountToSignInWithApple (ASCredentialServiceIdentifier serviceIdentifier, ASPasswordCredential existingCredential, [NullAllowed] NSDictionary userInfo);
+
+		[Export ("changePasswordWithoutUserInteractionForServiceIdentifier:existingCredential:newPassword:userInfo:")]
+		void ChangePasswordWithoutUserInteraction (ASCredentialServiceIdentifier serviceIdentifier, ASPasswordCredential existingCredential, string newPassword, [NullAllowed] NSDictionary userInfo);
+
+		[Export ("prepareInterfaceToChangePasswordForServiceIdentifier:existingCredential:newPassword:userInfo:")]
+		void PrepareInterfaceToChangePassword (ASCredentialServiceIdentifier serviceIdentifier, ASPasswordCredential existingCredential, string newPassword, [NullAllowed] NSDictionary userInfo);
```

Comment:
Same, WithApple seems redundant. Definitely not a reason to block the PR.

Reply:
same